pkg/cmd/projectconfig/add.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package projectconfig
+
+import (
+	"context"
+
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	"github.com/daytonaio/daytona/pkg/apiclient"
+	workspace_util "github.com/daytonaio/daytona/pkg/cmd/workspace/util"
+	"github.com/daytonaio/daytona/pkg/common"
+	"github.com/daytonaio/daytona/pkg/views"
+	"github.com/daytonaio/daytona/pkg/views/workspace/create"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var projectConfigAddCmd = &cobra.Command{
+	Use:   "add",
```

Comment:
`new` as an alias?

Reply:
Also, I think we should add the `--manual` flag to match `daytona create --manual`

Reply:
sure